### ISSUE TITLE
[StratCon] [49.19] Dual Enemy Mech Ace not working, or ARC2r ace not working

### ISSUE THREAD

**Issue by Tecmes** [MegaMek/mekhq#4099]
Got a scenario with two ace modifiers: ARC2R and FS9M.
In the lobby only the Firestarter was there. It was the second entry in the scenario, so maybe the Archer was overwritten??
The XML files *seem* similar.

**Comment by Thom293** [comment]
Yes.  I noticed this and forgot put up a bug report.  If 2 get drawn, only one will appear.  Happens with any combo of them.  I don't know why. I'll look at it but I think it's going to require coding.

**Comment by IllianiBird** [comment]
These scenario modifiers have been retired (or at least, aren't available by default). The reason this issue arose is because a single scenario cannot have duplicate modifiers and duplicate templates cannot be added to the same scenario either. I suspect the latter is the culprit here. Changing the internal template name should resolve the issue. Closing.
